sensor/tests/resource/service/service_test.go

```diff
@@ -117,6 +117,7 @@ type DeploymentExposureSuite struct {
 }
 
 func Test_DeploymentExposure(t *testing.T) {
+	t.Skip("Disabling these tests until we refactor the helper.go to enable fake k8s clients (ROX-13644)")
```

Comment:
Please also add a todo note in our standard format.

```suggestion
	// TODO(ROX-13644): reenable the test
	t.Skip("Disabling these tests until we refactor the helper.go to enable fake k8s clients (ROX-13644)")
```

Reply:
Done